### ISSUE TITLE
[More] returns 404

### ISSUE THREAD

**Issue by GoogleCodeExporter** [VBGAMER45/prettyurls#83]
```
What steps will reproduce the problem?
1. Reply to a topic
2. Have more smileys enabled
3. Click on [more]

What is the expected output? What do you see 
instead?
A popup with more smileys
404 error

What version of the mod, and what version of SMF are 
you using?
Rc1, 1.1.9

What other mods do you have installed?
None that effect that code


Please provide any additional information below.


```

Original issue reported on code.google.com by `elochaib...@gmail.com` on 24 Jun 2009 at 6:48

**Comment by GoogleCodeExporter** [comment]
```
Are you using the patched 1.0RC, or the original?
```

Original comment by `curiousdannii` on 27 Jun 2009 at 2:40

**Comment by GoogleCodeExporter** [comment]
```
Most likely was fixed in the 1.0RC patch.
```

Original comment by `curiousdannii` on 21 Jul 2009 at 5:17
- Changed state: **Fixed**
